### PR TITLE
Creating Flashing Troubleshooting article

### PR DESCRIPTION
**Pull request by jpwalden**
Signed-off-by: jari-pekka walden <jari-pekka.walden@jolla.com>

### CODE REVIEW

File: Support/Help_Articles/Flashing_Troubleshooting/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: Flashing Troubleshooting
+permalink: Support/Help_Articles/Flashing_Troubleshooting/
+parent: Help Articles
+layout: default
+nav_order: 50
+---
+
+#Flashing Sailfish to Xperia phone fails, troubleshooting.
```
Comment (vigejolla) [inline Support/Help_Articles/Flashing_Troubleshooting/README.md:9]: There should be a space between # and the heading. Same for all headings.

**Comment by jovirkku** [comment]
Line 13: "we use in on Lenovo Thinkpad" => "we use it on Lenovo Thinkpad"

**Review by jovirkku — APPROVED** [review]
Looks ok to me.

File: Support/Help_Articles/Flashing_Troubleshooting/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+1) Attach your Xperia to a USB port which is a **"USB root hub" (internal to your computer), to which no other device is attached (neither internally nor externally):
+* Execute lsusb in a terminal window (this is for Linux computers), without having your Xperia connected.
+* Look for a bus that solely has a Linux Foundation 2.0 root hub attached (i.e., nothing else).
+* Connect your Xperia to the USB port which is attached to this bus.
+* Execute lsusb again to make sure you selected the correct USB port; for example, an Xperia X on bus 003 then looks like this:
+```
+Bus 003 Device 015: ID 05c6:0afe Qualcomm, Inc. Xperia X Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+```
+
+2) Check if there is a **USB2 port \[1\]  on your computer or on the USB hub you are using. Connect your phone to the USB2 port for flashing.
```
Comment (vigejolla) [inline Support/Help_Articles/Flashing_Troubleshooting/README.md:54]:
```suggestion
2) Check if there is a **USB2 port [^1]  on your computer or on the USB hub you are using. Connect your phone to the USB2 port for flashing.
```

File: Support/Help_Articles/Flashing_Troubleshooting/README.md (hunk 1 above)
Comment (jpwalden) [reply Support/Help_Articles/Flashing_Troubleshooting/README.md]: Thanks, fixed

File: Support/Help_Articles/Flashing_Troubleshooting/README.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+3) If your USB hub has only USB3 ports, then connect the hub to the PC with a **USB2 data cable, and XA2 to the hub with any USB data cable ("USB C-type" in the XA2 end) \[2\].
```
Comment (vigejolla) [inline Support/Help_Articles/Flashing_Troubleshooting/README.md:56]:
```suggestion
3) If your USB hub has only USB3 ports, then connect the hub to the PC with a **USB2 data cable, and XA2 to the hub with any USB data cable ("USB C-type" in the XA2 end) [^2].
```

**Comment by jpwalden** [comment]
> Line 13: "we use in on Lenovo Thinkpad" => "we use it on Lenovo Thinkpad"

Thanks, fixed.

File: Support/Help_Articles/Flashing_Troubleshooting/README.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+## Can the Sailfish community help
+
+Our community has many talented persons, even experts. One of them has composed the following instructions for those struggling with Sailfish installation. 
+The article (read-only) is beyond this **[link](https://together.jolla.com/question/222126/)**.
+We/Jolla have not tested the tricks and methods of that article. They may or may not help. Following those instructions requires a fair amount of skills with Linux commands and computer systems, so we do not recommend them for beginners. Use them with care and at your own risk.
+
+You can also search help or send questions to the **[Forum](https://forum.sailfishos.org/)**.
+
+\[1\]  USB 3.0 Connectors are different from USB 2.0 connectors - they are usually (but not always) coloured blue on the inside in order to distinguish them from the 2.0 connectors. Check the specifications of your PC.
```
Comment (vigejolla) [inline Support/Help_Articles/Flashing_Troubleshooting/README.md:100]:
```suggestion
[^1]:  USB 3.0 Connectors are different from USB 2.0 connectors - they are usually (but not always) coloured blue on the inside in order to distinguish them from the 2.0 connectors. Check the specifications of your PC.
```

File: Support/Help_Articles/Flashing_Troubleshooting/README.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+\[2\]  Make sure that your USB cable is a **data cable (all 4 lines connected) and not just a charging cable. You can test this by copying a file over the cable from the PC to the phone or vice versa.
```
Comment (vigejolla) [inline Support/Help_Articles/Flashing_Troubleshooting/README.md:102]:
```suggestion
[^2]:  Make sure that your USB cable is a **data cable (all 4 lines connected) and not just a charging cable. You can test this by copying a file over the cable from the PC to the phone or vice versa.
```

**Review by vigejolla — COMMENTED** [review]
I suggest changing the footnotes to use markdown footnote syntax, otherwise LGTM

File: Support/Help_Articles/Flashing_Troubleshooting/README.md (hunk 2 above)
Comment (jpwalden) [reply Support/Help_Articles/Flashing_Troubleshooting/README.md]: Thanks, fixed

File: Support/Help_Articles/Flashing_Troubleshooting/README.md (hunk 3 above)
Comment (jpwalden) [reply Support/Help_Articles/Flashing_Troubleshooting/README.md]: Thanks, fixed

File: Support/Help_Articles/Flashing_Troubleshooting/README.md (hunk 4 above)
Comment (jpwalden) [reply Support/Help_Articles/Flashing_Troubleshooting/README.md]: Fixed

File: Support/Help_Articles/Flashing_Troubleshooting/README.md (hunk 5 above)
Comment (jpwalden) [reply Support/Help_Articles/Flashing_Troubleshooting/README.md]: Fixed

**Review by vigejolla — APPROVED** [review]

**Pull request merged**